### PR TITLE
CONTROLLERS:  API Security Tests

### PR DESCRIPTION
**Pull request by cjdutoit**
CONTROLLERS:  API Security Tests

### CODE REVIEW

File: ISL.ReIdentification.WebApps.Configurations/ISL.ReIdentification.Configurations.Server.Tests.Unit/Controllers/AuditAcesses/AccessAuditsControllerTests.Security.Delete.cs [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+namespace ISL.ReIdentification.Configurations.Server.Tests.Unit.Controllers.AccessAudits
+{
+    public partial class AccessAuditsControllerTests
+    {
+        [Fact]
+        public void DeleteShouldHaveRoleAttributeWithRoles()
+        {
+            // Given
+
```
[the remark this answers is not in the document]
Comment (cjdutoit) [reply ISL.ReIdentification.WebApps.Configurations/ISL.ReIdentification.Configurations.Server.Tests.Unit/Controllers/AuditAcesses/AccessAuditsControllerTests.Security.Delete.cs]: done

[the remark this answers is not in the document]
Comment (cjdutoit) [reply ISL.ReIdentification.WebApps.Configurations/ISL.ReIdentification.Configurations.Server.Tests.Unit/Controllers/AuditAcesses/AccessAuditsControllerTests.Security.Delete.cs]: done

[the remark this answers is not in the document]
Comment (cjdutoit) [reply ISL.ReIdentification.WebApps.Configurations/ISL.ReIdentification.Configurations.Server.Tests.Unit/Controllers/AuditAcesses/AccessAuditsControllerTests.Security.Delete.cs]: done

File: ISL.ReIdentification.WebApps.Portals/ISL.ReIdentification.Portals.Server.Tests.Unit/Controllers/PdsDatas/PdsDataControllerTests.Logic.Delete.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            PdsData expectedPdsData = storagePdsData.DeepClone();
+
+            var expectedObjectResult =
+                new OkObjectResult(expectedPdsData);
+
+            var expectedActionResult =
+                new ActionResult<PdsData>(expectedObjectResult);
+
+            pdsDataServiceMock
+                .Setup(service => service.RemovePdsDataByIdAsync(It.IsAny<Guid>()))
```
[the remark this answers is not in the document]
Comment (cjdutoit) [reply ISL.ReIdentification.WebApps.Portals/ISL.ReIdentification.Portals.Server.Tests.Unit/Controllers/PdsDatas/PdsDataControllerTests.Logic.Delete.cs:33]: Fixed

**Review by JParkesAD — APPROVED** [review]

**Pull request merged**